docs/build/isc/v1.1/docs/how-tos/core-contracts/nft/get-L2-nfts.mdx

```diff
@@ -0,0 +1,97 @@
+---
+description: How to get the L2 NFTs owned by an account.
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - EVM
+  - how-to
+  - native tokens
+  - L2 NFTs
+---
+import GetNftMetadata from '../../../_partials/how-tos/token/_get-nft-metadata.md';
+
+# Get NFTs Owned by an Account
+
+This guide will show you how to get the L2 NFTs owned by an account using the [`ISCAccounts.getL2NFTs`](../../../reference/magic-contract/ISCAccounts.md#getl2nfts) function from the [`ISCAccounts.getL2NFTs`](../../../reference/magic-contract/ISCAccounts.md) [magic contract](../../../reference/magic-contract/introduction.md).
+
+<GetNftMetadata />
+
+## Understanding the `getL2NFTs` Function
+
+The `getL2NFTs` function is a view function provided by the `ISCAccounts` interface. It takes an `ISCAgentID` as an argument and returns an array of `NFTID` objects. This function allows users to query and retrieve the list of L2 NFTs owned by a specified agent. The `ISCAgentID` represents the identifier of the agent (user or contract) whose NFTs you want to retrieve. The returned array contains the identifiers of the NFTs, which can then be used to fetch more details about each NFT if needed.
```

Comment:
Maybe we should add an admonition here on what an AgentID exactly is and link to more content.

We should probably also add How-Tos on how to do the most common type conversions. Like address to agentid or NFTID to token and so on

Reply:
@Dr-Electron, I added an admonition, but I don't know if the info about ISCAgentID is enough. I think the URL I referenced in the admonition might not be the best.

Reply:
Or maybe you want me to remove this enter section and just leave the admonition?

Reply:
It is good enough for now. Lets see if we want to change it later or extend the linked article in another iteration